### PR TITLE
[NIL-99] Adds Indicator document type to search results

### PR DESCRIPTION
**Pull request by hugosykes**
[NIL-99] Adds Indicator document type to left panels of facets.
Updates national indicator library sitemap to render national indicators hub page instead of blank page for root directory (/site/indicators).

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/document-types.yaml [hunk 1]
```diff
@@ -15,24 +15,27 @@
...
     - ''
     - ''
+    - ''
     resourcebundle:id: document-types
     resourcebundle:keys:
     - publicationsystem:archive
     - publicationsystem:series
     - publicationsystem:dataset
     - publicationsystem:publication
+    - nationalindicatorlibrary:indicator
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/document-types.yaml]: I think this bundle is no longer in use and can be deleted.

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/facet-labels.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    jcr:primaryType: resourcebundle:resourcebundle
+    resourcebundle:descriptions:
+    - ''
+    - ''
+    - ''
+    - ''
+    - ''
+    resourcebundle:id: facet-labels
+    resourcebundle:keys:
+    - facet.archive
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/facet-labels.yaml]: this is my fault, but you don't need this label (archive) @hugosykes

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/headers.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+/content/documents/administration/national-indicator-library/headers:
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/headers.yaml]: this looks like it needs `make format-yaml`

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/labels.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    hippostdpubwf:lastModifiedBy: admin
+    jcr:mixinTypes:
+    - mix:referenceable
+    jcr:primaryType: resourcebundle:resourcebundle
+    resourcebundle:descriptions:
+    - ''
+    - ''
+    resourcebundle:id: nationalindicatorlibrary.labels
+    resourcebundle:keys:
+    - labels.indicator
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/labels.yaml]: not used, please remove

File: repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/labels.yaml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    - labels.nil-search-ghost-text
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/labels.yaml]: not used, please remove

File: repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/national-indicator-library/sample-indicator.yaml [hunk 6]
```diff
@@ -66,30 +67,29 @@
...
-    nationalindicatorlibrary:purpose: Patients who have had a stroke should be admitted
-      directly to a specialist acute stroke unit. Getting patients to a stroke unit
-      quickly is a strong indicator of eventual outcomes and is also closely linked
-      to improved quality of care across other stroke care markers. The indicator
-      is therefore an important measure of the effectiveness of recognition and assessment
-      of the symptoms of stroke, and the process to transfer people to a stroke unit
-      in a timely manner. Patients who have a stroke in hospital are included in the
-      indicator to take into account the process of recognising the stroke has occurred
-      and the systems in place for in-hospital pathways.
+    nationalindicatorlibrary:publishedBy: Publisher directly to a specialist acute
```
Comment (wojtek-oledzki) [inline repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/national-indicator-library/sample-indicator.yaml]: looks like you've merged "purpose" with "publishedBy"

File: repository-data/webfiles/src/main/resources/site/freemarker/common/facets.ftl [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <#ftl output_format="HTML">
 <#include "../include/imports.ftl">
 <!--Need to have a single setBundle call as subsequent ones will overwrite the previous values-->
-<@hst.setBundle basename="document-types,month-names,publicationsystem.labels,facet-headers"/>
+<@hst.setBundle basename="document-types,month-names,publicationsystem.labels,nationalindicatorlibrary.labels,facet-headers,facet-labels"/>
```
Comment (wojtek-oledzki) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/facets.ftl]: no need for `nationalindicatorlibrary.labels` and I think `publicationsystem.labels` are no longer used as well

File: repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/folder.ftl [hunk 8]
```diff
@@ -7,6 +7,8 @@
     <#include "./series.ftl">
 <#elseif publication??>
     <#include "./publication.ftl">
+<#elseif indicator??>
```
Comment (wojtek-oledzki) [inline repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/folder.ftl]: ???

File: site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java [hunk 9]
```diff
@@ -182,6 +186,13 @@ private void addPublicationSystemTypes(HstQueryBuilder query) {
         );
     }
 
+    /**
+     * Indicatoir ....
```
Comment (wojtek-oledzki) [inline site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: tiny bit of comment in code please

File: site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java [hunk 10]
```diff
@@ -192,4 +203,14 @@ private Constraint publicationSystemConstraint(String query) {
             constraint("publicationsystem:KeyFacts").contains(query)
         );
     }
+
+    /**
+     * ...
```
Comment (wojtek-oledzki) [inline site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java]: tiny bit of comment in code please

File: repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl [hunk 11]
```diff
@@ -10,7 +10,6 @@
   <meta charset="UTF-8" />
   <meta name="title" content="NHS - National Indicator Library" />
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0" />
-  <link rel="stylesheet" href="css/style.css" />
```
Comment (wojtek-oledzki) [inline repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl]: I will check with you tomorrow why you're not using "base-layout" tpl

Comment (hugosykes) [reply repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl]:
```yaml
/nationalindicatorlibrary:
        hst:inheritsfrom:
        - ../base
        jcr:primaryType: hst:configuration
```

Comment (hugosykes) [reply repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl]: Still inheriting from base, this includes the base-layout template doesn't it? I just noticed that if I took out that stylesheet reference, it made no difference so I removed it

**Pull request merged**